checkDLomix.txt

```diff
@@ -0,0 +1,19 @@
+ 	sequence	irt	predicted	diff
+0	    SSAAPPPPPR	33.18	39.65256	-6.47256
+1	  GISNEGONASIK	61	38.428078	22.571922
+2	  HVLTSIGEK	39.45	43.108276	-3.658276
+3	DIPVPKPK	47.28	39.879616	7.400384
+4	LSSEAPALFQFDLK	68.48	98.93983	-30.45983
+5	IGDYAGIK	49.46	48.897575	0.562425
+6	 TASEFDSAIAQDK	56.58	57.383064	-0.803064
+7	  SAAGAFGPELSR	61.02	63.235287	-2.215287
+8	ELGQSGVDTYLQTK	66.73	72.97594	-6.24594
+9	GLILVGGYGTR	78.15	79.66746	-1.51746
+10	GILFVGSGVSGGEEGAR	79.32	83.234665	-3.914665
+11	SFANQPLEVVYSK	78.2	82.19065	-3.99065
+12	  LTILEELR	88.05	87.32299	0.72701
+13	NGFILDGFPR	92.87	96.25705	-3.38705
+14	ELASGLSFPVGFK	96.19	98.41434	-2.22434
+15	  LSSEAPALFQFDLK  	101.78	98.93983	2.84017
+16	 GISNEGQNASIK	36.7	37.439228	-0.739228
+
```

Comment:
not clear if this is a test output file from you run, but it should be pushed to the repo